adhocracy4/comments_async/serializers.py

```diff
@@ -148,9 +148,9 @@ class CommentListSerializer(CommentSerializer):
 
     def get_comment(self, obj):
         if obj.is_removed:
-            return _('deleted by creator')
+            return _('')
```

Comment:
And if you would want to change it, there is no need to translate an empty string. :)